src/components/frame/Extensions/layouts/PageLayout.tsx

```diff
@@ -45,42 +46,44 @@ const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({
   }, [isDarkMode]);
 
   return (
-    <Spinner loading={false} loadingText={loadingText}>
-      {canUpgrade && <CalamityBanner items={calamityBannerItems} />}
-      <ToastContainer
-        className={styles.toastNotification}
-        autoClose={3000}
-        hideProgressBar
-        closeOnClick
-        rtl={false}
-        pauseOnFocusLoss
-        pauseOnHover
-        closeButton={CloseButton}
-      />
-      <UserTransactionContextProvider>
-        <Header />
-      </UserTransactionContextProvider>
-      {/* @TODO: Remove wallet component when we have a proper wallet */}
-      <div className="hidden">
-        <Wallet />
-      </div>
-      <main className="mt-5 pb-24">
-        <div className="inner">
-          {isMobile && (
-            <div className="mb-9">
-              <Navigation pageName={pageName} />
-            </div>
-          )}
-          <PageTitle title={title} subtitle={description} />
-          <div className="mt-9">{children}</div>
+    <TokensModalContextProvider>
+      <Spinner loading={false} loadingText={loadingText}>
+        {canUpgrade && <CalamityBanner items={calamityBannerItems} />}
```

Comment:
Have you checked how does it look like when CalamityBanner and Toast are both visible?

Reply:
![image](https://github.com/JoinColony/colonyCDapp/assets/76940796/826242c5-457e-48b0-a2ca-6ab2a7f2030e)